pkgs/sdk/server/test/Internal/DataSources/FileDataSourceTest.cs

```diff
@@ -148,9 +151,52 @@ public void ModifiedFileIsReloadedIfAutoUpdateIsOn()
 
                     file.SetContentFromPath(TestUtils.TestFilePath("segment-only.json"));
 
-                    var newData = _updateSink.Inits.ExpectValue(TimeSpan.FromSeconds(5));
-
-                    AssertJsonEqual(DataSetAsJson(ExpectedDataSetForSegmentOnlyFile(2)), DataSetAsJson(newData));
```

Comment:
Cannot just do a JSON comparison if the version number is going to be incremented every time a change is detected.

Reply:
This isn't pretty, but I want the test to stop failing.